aws/sdk/examples/config/src/bin/show-resource-history.rs

```diff
@@ -1,77 +1,87 @@
 use aws_types::region;
 use aws_types::region::ProvideRegion;
 use config::model::ResourceType;
-use config::{Client, Config, Error, Region};
+use config::{Client, Config, Error, Region, PKG_VERSION};
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 struct Opt {
-    /// The default AWS Region.
+    /// The AWS Region.
     #[structopt(short, long)]
-    default_region: Option<String>,
+    region: Option<String>,
 
-    /// The resource id.
-    #[structopt(long)]
-    resource_id: String,
+    /// The ID of the resource.
+    #[structopt(short, long)]
+    id: String,
 
-    /// The resource type, eg. "AWS::EC2::SecurityGroup"
-    #[structopt(long)]
-    resource_type: String,
+    /// The resource type.
+    #[structopt(short, long)]
+    type_: String,
```

Comment:
since `type` is a keyword in Rust, I'd call this `resource_type` as it was before but remove the short option

Reply:
Done